src/scheduler/workers/unpublish/day/index.ts

```diff
@@ -0,0 +1,19 @@
+import { logger } from "server/services/logger";
+import { findListsInAnnualReviewForReminders } from "../findListsInAnnualReviewForReminders";
+import { changeState } from "scheduler/workers/unpublish/day/changeState";
+import { findListsToResetAnnualReview } from "scheduler/workers/unpublish/day/changeState/findListsToResetAnnualReview";
+import { sendUnpublishEmails } from "scheduler/workers/unpublish/day/sendUnpublishEmails";
+
+export async function main() {
+  const listsInAnnualReview = await findListsInAnnualReviewForReminders();
+  const results = await Promise.allSettled([listsInAnnualReview.map(sendUnpublishEmails)]);
```

Comment:
Again, I could be wrong but here `listsInAnnualReview.map(sendUnpublishEmails)` the map method will return an array, does it make sense to wrap this in an array as well? Would `Promise.allSettled` work with a nested array?